cmd/bnscli/cmd_gov.go

```diff
@@ -1,112 +1,84 @@
 package main
 
 import (
-	"encoding/binary"
+	"errors"
 	"flag"
 	"fmt"
 	"io"
+	"io/ioutil"
 	"time"
 
 	"github.com/iov-one/weave"
 	"github.com/iov-one/weave/cmd/bnsd/app"
-	"github.com/iov-one/weave/coin"
 	"github.com/iov-one/weave/x/cash"
+	"github.com/iov-one/weave/x/distribution"
 	"github.com/iov-one/weave/x/escrow"
 	"github.com/iov-one/weave/x/gov"
 )
 
-func cmdNewTransferProposal(input io.Reader, output io.Writer, args []string) error {
+func cmdAsProposal(input io.Reader, output io.Writer, args []string) error {
 	fl := flag.NewFlagSet("", flag.ExitOnError)
 	fl.Usage = func() {
 		fmt.Fprintln(flag.CommandLine.Output(), `
-Create a proposal transaction for transfering funds from source account to the destination.
+Read a transaction from the stdin and extract message from it. create a
+proposal transaction for that message. All attributes of the original
+transaction (ie signatures) are being dropped.
 		`)
 		fl.PrintDefaults()
 	}
 	var (
-		srcFl    = flAddress(fl, "src", "", "A source account address that the founds are send from.")
-		dstFl    = flAddress(fl, "dst", "", "A destination account address that the founds are send to.")
-		amountFl = flCoin(fl, "amount", "1 IOV", "An amount that is to be transferred between the source to the destination accounts.")
-		memoFl   = fl.String("memo", "", "A short message attached to the transfer operation.")
-		titleFl  = fl.String("title", "Transfer funds to distribution account", "The proposal title.")
-		descFl   = fl.String("description", "Transfer funds to distribution account", "The proposal description.")
-		eRuleFl  = fl.Uint64("electionrule", 0, "The ID of the election rule to be used.")
+		titleFl = fl.String("title", "Transfer funds to distribution account", "The proposal title.")
+		descFl  = fl.String("description", "Transfer funds to distribution account", "The proposal description.")
+		eRuleFl = fl.Uint64("electionrule", 0, "The ID of the election rule to be used.")
```

Comment:
Nevermind, I see you do have the flag, just not in the demo code.
I think there is an author field too. Shall we make that possible to add as well?